src/stk500.h

```diff
@@ -57,6 +57,9 @@ struct pdata {
   bool fosc_get;
   bool fosc_set;
   double fosc_data;
+
```

Comment:
Perhaps neater to have inline comment at column 32 without empty line? So adding a component is one line, not three.